components/hook-http-server/node/index.mjs

```diff
@@ -176,6 +178,14 @@ export default (dependencies) => {
     return tracking;
   };
 
+  const forwardTraffic = (
+    _state,
```

Comment:
`_state` intended to be unused?

Reply:
Yes it is because I need the same type signature as `spyTraffic`

```js
const handleTraffic = http ? spyTraffic : forwardTraffic;
```